### ISSUE TITLE
Use case of detection of multi words and sentences in a image

### ISSUE THREAD

**Issue by mit456** [meijieru/crnn.pytorch#53]
Hello,

I was googling into current research on OCR and scene detection using a deep neural network, came across CRNN and STN-OCR.

I tried `demo.py` given in this project, which works well on a single word image but for images which has multiple words/sentence then it's kind on failing maybe because trained model is not meant for that.

I am thinking if I train CRNN on msynth data will it help solving my use case?  Actually, msynth data only contains single word images and labels, So I am being doubtful whether will it help or not.

**Comment by meijieru** [comment]
This method is designed for single word because CTC do not distinguish between a true blank and a blank for CTC. 
For text line recognition you may want some other method.

**Comment by mit456** [comment]
@meijieru Thanks for the reply. 

It would be grateful if you can give me some kind of path, I mean if any research paper/implementation I should look at.

**Comment by meijieru** [comment]
I am not familiar with OCR, so it would be better to search other place.

**Comment by rohun-tripathi** [comment]
I am using the CRNN for Handwritten OCR for one word. I am using the IAM dataset to train the net. Have you tried this variation @mit456 ? Have your tried a pipeline where the words are identified and segmented first and then followed up by this model? I was planning to use this.
